olm-catalog/serverless-operator/Dockerfile

```diff
@@ -13,7 +13,7 @@ LABEL operators.operatorframework.io.bundle.channels.v1="4.6"
 LABEL \
```

Comment:
The only issues I can see is with the channels. I think we'll want 4.7 as well. But that can probably be fixed later

Reply:
Adding 4.7?  or replacing 4.6 with 4.7 ? 

I am fine w/ a follow up PR on this, tbh

Reply:
Should definitely be discussed separately.